### ISSUE TITLE
add `--no-capture-output` to run command

### ISSUE THREAD

**Issue by harelwa** [OldGrumpyViking/hatch-conda#13]
hi,

some commands ran with `hatch run [...]` require printing the command output. 

for example, running a docs server (e.g. `mkdocs serve`) requires the URL ..

as there `--no-capture-output` is not provided to the **run** command, these outputs are hidden

if this is agreed, i'd be happy to contribute a PR

Thanks,

Harel

**Comment by OldGrumpyViking** [comment]
Hi the run command runs without the capture_output set to True and is default False.
Is this not the behavior that you wish? Please elaborate if i am misunderstanding.

https://docs.python.org/3/library/subprocess.html#subprocess.run

**Comment by harelwa** [comment]
the flag is a boolean flag. when absent, it captures the output. this is not good for us, as we do want the output to show up in the console. the example above (mkdocs serve) is just one and very common. currently output is hidden.

**Comment by OldGrumpyViking** [comment]
When i run a command i see the output. So i might be misunderstanding what you mean. You are welcome to make a PR.

```
(base) C:\Users\XXX\git\python-template>hatch run conda:pip -V
pip 23.3.2 from C:\Users\XXX\AppData\Local\miniconda3\envs\python-template_conda_3.8\lib\site-packages\pip (python 3.8)
```

**Comment by OldGrumpyViking** [comment]
Aha, i see the output is displayed when the command finishes and as the mkdocs server command does not finish the output is not emitted.

Thanks for the PR 👍🏻

**Comment by harelwa** [comment]
solved in #20
